### PR TITLE
chore(Java): Kafka Connect

### PR DESCRIPTION
**Pull request by meiao**
## Give us some context
Documentation for a new feature in the Java agent.

### CODE REVIEW

**Review by rhetoric101 — COMMENTED** [review]
@meiao thanks for making these additions! I inserted some questions below, and I also just pushed up a change to that long list of examples for metrics. Could you take a look at my changes to confirm?

File: src/content/docs/apm/agents/java-agent/getting-started/compatibility-requirements-java-agent.mdx [hunk 1]
```diff
@@ -442,10 +442,12 @@ The agent automatically instruments these frameworks and libraries:
...
-    * [Kafka Clients](/docs/agents/java-agent/instrumentation/use-kafka-message-queues) 0.11.0.0 to latest (for distributed tracing and event data)
-    * [Kafka Streams](/docs/agents/java-agent/instrumentation/use-kafka-message-queues) 1.0.0 to latest (for metrics)
-    * [Kafka Streams](/docs/agents/java-agent/instrumentation/use-kafka-message-queues) 2.0.0 to latest (for distributed tracing and transaction data)
+    * [Kafka Clients](/docs/agents/java-agent/instrumentation/java-agent-instrument-kafka-message-queues) 0.10.0.0 to latest (for metric and event data)
+    * [Kafka Clients](/docs/agents/java-agent/instrumentation/java-agent-instrument-kafka-message-queues) 0.11.0.0 to latest (for distributed tracing and event data)
+    * [Kafka Connect](/docs/agents/java-agent/instrumentation/java-agent-instrument-kafka-message-queues) 1.0.0 to latest (for metrics)
+    * [Kafka Connect](/docs/agents/java-agent/instrumentation/java-agent-instrument-kafka-message-queues) 2.0.0 to latest (for distributed tracing and transaction data)
+    * [Kafka Streams](/docs/agents/java-agent/instrumentation/java-agent-instrument-kafka-message-queues) 1.0.0 to latest (for metrics)
```
[the remark this answers is not in the document]
Comment (meiao) [reply src/content/docs/apm/agents/java-agent/getting-started/compatibility-requirements-java-agent.mdx:449]: Not sure which line you are referring to (that duplicates 447). But there are a bunch of lines in there, and pretty much the only difference between them is the beginning:
- Kafka Clients
- Kafka Connect
- Kafka Streams

File: src/content/docs/apm/agents/java-agent/getting-started/compatibility-requirements-java-agent.mdx [hunk 2]
```diff
@@ -442,10 +442,12 @@ The agent automatically instruments these frameworks and libraries:
...
+    * [Kafka Streams](/docs/agents/java-agent/instrumentation/java-agent-instrument-kafka-message-queues) 2.0.0 to latest (for distributed tracing and transaction data)
```
[the remark this answers is not in the document]
Comment (meiao) [reply src/content/docs/apm/agents/java-agent/getting-started/compatibility-requirements-java-agent.mdx:450]: Same here

File: src/content/docs/apm/agents/java-agent/getting-started/compatibility-requirements-java-agent.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (meiao) [reply src/content/docs/apm/agents/java-agent/getting-started/compatibility-requirements-java-agent.mdx:449]: In the diff view I believe you hightlighted 449, and yes, 447 is Kafka Connect and 449 it is Kafka Streams.

File: src/content/docs/apm/agents/java-agent/getting-started/compatibility-requirements-java-agent.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (rhetoric101) [reply src/content/docs/apm/agents/java-agent/getting-started/compatibility-requirements-java-agent.mdx:450]: This was my mistake! These are separate entries for Connect and Streams!

**Review by rhetoric101 — APPROVED** [review]
@meiao thanks for making these updates!

**Pull request merged**